features/machine/machine.feature

```diff
@@ -504,6 +509,7 @@ Feature: Machine features testing
   @admin
   @destructive
   @4.9
+  @vsphere-ipi
```

Comment:
We may need to add the tag to specific `Examples` lines, as it contains scenario for several iaas there.

Reply:
updated the tags